### PR TITLE
test(sumologicexporter): Add tests for `metadata_attributes`

### PR DESCRIPTION
**Pull request by astencel-sumo**
These tests specify how the `metadata_attributes` configuration works for different formats of logs and metrics.

### CODE REVIEW

File: pkg/exporter/sumologicexporter/sender_test.go [hunk 1]
```diff
@@ -60,6 +60,7 @@ func prepareSenderTest(t *testing.T, cb []func(w http.ResponseWriter, req *http.
 		MetricFormat:       "carbon2",
 		Client:             "otelcol",
 		MaxRequestBodySize: 20_971_520,
+		GraphiteTemplate:   DefaultGraphiteTemplate,
```
Comment (pmalek-sumo) [inline pkg/exporter/sumologicexporter/sender_test.go:63]: Why is this needed here? Maybe we should use the default config via `createDefaultConfig` and then change it here?

**Comment by sumo-drosiek** [comment]
Is adding new test file with unit tests which tests another file a good approach? I mean, now we will have two unit test files for exporter: `exporter_metadata_test.go` and `exporter_test.go`

**Comment by astencel-sumo** [comment]
> Is adding new test file with unit tests which tests another file a good approach? I mean, now we will have two unit test files for exporter: `exporter_metadata_test.go` and `exporter_test.go`

Right, thanks for bringing this to attention. I had a feeling it's hard to find anything in such a single big file with unit tests and thought maybe splitting into a couple separate files could help.

File: pkg/exporter/sumologicexporter/sender_test.go (hunk 1 above)
Comment (astencel-sumo) [reply pkg/exporter/sumologicexporter/sender_test.go:63]: I agree it would be more readable to use the default config and override anything needed. I'll refactor that.

Comment (astencel-sumo) [reply pkg/exporter/sumologicexporter/sender_test.go]: Done @pmalek-sumo please take a look

**Comment by astencel-sumo** [comment]
> > Is adding new test file with unit tests which tests another file a good approach? I mean, now we will have two unit test files for exporter: `exporter_metadata_test.go` and `exporter_test.go`
> 
> Right, thanks for bringing this to attention. I had a feeling it's hard to find anything in such a single big file with unit tests and thought maybe splitting into a couple separate files could help.

@sumo-drosiek I have moved the new tests into `exporter_test.go` to keep all tests in single file. Plese take a look.

**Review by pmalek-sumo — APPROVED** [review]
